DATAFLOW.md

```diff
@@ -1,28 +1,27 @@
 # Enterprise Knowledge Solution (EKS) Dataflow
 
 1. **Upload documents and trigger document processing workflow**
-   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook, .xlsx, .pdf, .docx) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
-
-   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.
-
+   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
+   
+   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.  
 1. **Validate the documents:**
    Documents that fail validation are redirected to a separate "reject" bucket within Google Cloud Storage for further review and handling.
 1. **Initiate Processing Jobs:**
    The workflow initiates specific processing jobs based on the document type:
-   `PDF Classifier (Cloud Run -> Doc AI Custom Classifier):` Identify PDF documents that contain forms.
-   `Forms Processor (Cloud Run -> Doc AI Form Parser):` Processes form documents(PDF).
-   `Excel, Outlook Processor (Cloud Run):` Processes Excel and Outlook files.
-1. **Extract and store content:**
-   Each processor extracts relevant content from the documents. The extracted content (PDF, docx, txt) is stored in a “process” bucket within Google Cloud Storage.
-1. **Create and store document metadata for Excel and Outlook documents:**
-   The extracted document metadata is stored in a BigQuery table.
+   - `Document Classifier (Cloud Run -> Doc AI Custom Classifier):`  Identify PDF document type (Any form, bank statements, invoices, pay slip, lending doc and utility bills)  
+   - `Specialized Parser (Cloud Run -> Doc AI Form and Specialied Parser):` Processes forms and documents (Any form, bank statements, invoices, pay slip, lending doc and utility bills)  
+   - `Excel, Outlook Processor (Cloud Run):` Processes Excel and Outlook files.  
 1. **Utilize Document AI Batch API:**
-   The Document Processing Workflow leverages the Document AI Batch API for asynchronous processing of large document volumes.
-1. **Create and store document metadata for forms in PDF documents:**
-   The extracted document metadata is stored in a BigQuery table.
+   The Specialized Parser Job leverages Document AI Batch API for asynchronous processing of large document volumes.
+1. **Extract and store data elements:**
```

Comment:
It's hard to follow how steps 5, 6, and 7 are different. They're saying mostly the same thing but highlight different data products, but it's not clear to the reader why the multiple sources are required or what outcomes are enabled by the multiple sources